src/data/data1d/filter1d/fft.js

```diff
@@ -50,7 +50,7 @@ function generateXAxis(datum1D) {
   const info = datum1D.info;
   const baseFrequency = parseFloat(info.bf1);
   const spectralFrequency = parseFloat(info.sfo1);
-  const spectralWidth = parseFloat(info.sw);
+  const spectralWidth = parseFloat(info['spectral width']);
```

Comment:
All the property names in info should be camelCase. Please rename it to spectralWidth